core/sql/regress/compGeneral/EXPECTED023

```diff
@@ -23,6 +23,10 @@ SB_HISTOGRAMS
 SB_HISTOGRAM_INTERVALS
 SB_PERSISTENT_SAMPLES
 
+=======================
```

Comment:
Would it be better to change the GET output like SELECT where we say "n row(s) returned" where n >=0

Reply:
I agree, this looks much better!

Reply:
Let me change it step by step.

Reply:
Agree with Venkat. The GET statement should behave like a SELECT as much as possible. Ideally, we would someday allow it in places where we allow select:

```
select *
from (get users union all get schemas) T(o)
where o like 'finance%'
```

I know, that's science fiction for now, but it would be good to make sure this will some day be possible.

Reply:
This is not that much of science fiction. 
get commands can be used as derived tables within selects.

For ex: one can do the following to get this result:
>>create table if not exists "finance" (a int);

--- SQL operation complete.
>>select left(b, 10) from
+>((select * from (get tables in schema sch) x(a))
+> union all
+> (select * from (get users) x(a))) z(b)
+>where b like 'finance%';

(EXPR)                                  
----------------------------------------

finance                                 

--- 1 row(s) selected.
>>

Reply:
Very cool, thanks!